modules/openapi-json-schema-generator/src/main/resources/python/endpoint.handlebars

```diff
@@ -543,8 +567,20 @@ class {{operationIdCamelCase}}(BaseApi):
 class ApiFor{{httpMethod}}(BaseApi):
     # this class is used by api classes that refer to endpoints by path and http method names
 
+    @typing.overload
+    def {{httpMethod}}(
+    {{> endpoint_args isOverload=true skipDeserialization="False"}}
+
+    @typing.overload
+    def {{httpMethod}}(
+    {{> endpoint_args isOverload=true skipDeserialization="True"}}
```

Comment:
For this 3rd one, can you update the generated input signature `skip_deserialization` to be:
`skip_deserialization:  bool` without a default value?
Have you checked pycharm to make sure that these are working correctly when passed:
- skip_deserialization True
- skip_deserialization False
- skip_deserialization unset
- skip_deserialization arbitrary boolean

Reply:
Didn't use pycharm.
VSCode type checker (Pylance) will complain Non-default argument follows default argument.
And four situations you mentioned looks work right on my environment.

Reply:
With my boolean suggestion without a default the argument will have to be moved earlier because it has no default. Can you try making that change and seeing if it works? If it does then it will achieve the same result with less code.